modules/apps/permission-template/source-formatter-suppressions.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+
+<suppressions>
+	<source-check>
+		<suppress checks="XMLServiceEntityNameCheck" files="permission-template-service/service\.xml" />
+	</source-check>
+</suppressions>
```

Comment:
I think this should be committed under [5984416](https://github.com/tototrinh/liferay-portal/pull/83/commits/598441604cfe2cc0b77e1acd1d55e01cb11b5065)

Reply:
Fixed it. Thanks